src/reducers/inputs.js

```diff
@@ -65,33 +65,53 @@ export const removeAllContent = (state, action) => {
   return validate(newState)
 }
 
+/**
+ * Takes the reduxPath (an array of keys that correspond to the redux state tree for a 'resource') and performs a reduce
+ * function on each of the keys, searching for the last key in the path and then appending an object with an `items` array.
+ * Also checks for needed blank nodes along the reduxPath in the state tree and appends intermediate objects.
+ *
+ * @returns a {function} with {{}}
```

Comment:
Should this be:

```javascript
 * @returns {function} a function with? that... ?
```

Reply:
Thanks. Not really sure how to write these things...LMK if it is better...